source/fundamentals/crud/read-operations/project.txt

```diff
@@ -0,0 +1,169 @@
+==============================
+Specify Which Fields to Return
+==============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to specify whether fields in a
+document should return from read operations.
+
+Sample Data
+~~~~~~~~~~~
+
+Run the following snippet to load the documents into the ``ratings``
+collection of the ``tea`` database:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/projection.go 
+   :language: go
+   :dedent:
+   :start-after: begin insertDocs
+   :end-before: end insertDocs
+
+.. include:: /includes/fundamentals/tea-sample-data-ending.rst
+
+Project
```

Comment:
[question]

This is the first time we're mentioning the idea of "projection" on this page. Up until now, we've been saying "specify which fields to return". Do you think we need a sentence explaining that projection _is_ specifying which fields to return?

If I'm a user who has no idea what projection is, will I be confused without seeing a definition of projection? I'm not sure this is 100% necessary, but want to raise it for consideration.